lxd/patches.go

```diff
@@ -70,6 +70,7 @@ var patches = []patch{
 	{name: "warnings_remove_empty_node", stage: patchPostDaemonStorage, run: patchRemoveWarningsWithEmptyNode},
 	{name: "dnsmasq_entries_include_device_name", stage: patchPostDaemonStorage, run: patchDnsmasqEntriesIncludeDeviceName},
 	{name: "storage_missing_snapshot_records", stage: patchPostDaemonStorage, run: patchGenericStorage},
+	{name: "clean_storage_volumes", stage: patchPostDaemonStorage, run: patchGenericStorage},
```

Comment:
Lets align this with the earlier patch naming, so `storage_delete_old_snapshot_records`